server/controllers/tasks/tasks.ts

```diff
@@ -25,8 +25,10 @@ export default {
       'deduplicate-works': deduplicateWorks,
     },
     admin: {
-      'collect-entities': collectEntities,
-      'check-entities': checkEntities,
+      // Untested endpoint which triggers a global check of local database entities,
```

Comment:
It has been tested, just not automatically. The current wording might make it sound like it has never even been tried ^^'
```suggestion
      // Endpoint with no automatic tests coverage that triggers a global check of local database entities,
```

Reply:
done in e620dc3691